lib/Maintenance/Tasks/TaskHelper.php

```diff
@@ -0,0 +1,75 @@
+<?php
+declare(strict_types=1);
+
+namespace Pimcore\Maintenance\Tasks;
+
+use Pimcore\Db;
+use Pimcore\Model\DataObject\ClassDefinition;
+use Psr\Log\LoggerInterface;
+
+final class TaskHelper
+{
+    private static ?TaskHelper $instance = null;
+
+    private LoggerInterface $logger;
+    public static function create(LoggerInterface $logger): self
+    {
+        if (!self::$instance) {
+            self::$instance = new self();
+        }
+        self::$instance->logger = $logger;
+
+        return self::$instance;
+    }
+    
+    public function getDataStructureNamesMapLowerToCamelCase(string $dir): array
```

Comment:
```suggestion
    public function getCollectionNames(string $dir): array
```
I don't see any camel case conversion.